### PR TITLE
CIS Range app development

### PR DESCRIPTION
**Pull request by KumarGanesanIBM**

### CODE REVIEW

File: ibm/resource_ibm_cis_range_app.go [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	ibmCISRangeApp                     = "ibm_cis_range_app"
+	cisRangeAppID                      = "app_id"
+	cisRangeAppProtocol                = "protocol"
+	cisRangeAppDNS                     = "dns"
+	cisRangeAppDNSType                 = "dns_type"
+	cisRangeAppDNSName                 = "name"
```
Comment (kavya498) [inline ibm/resource_ibm_cis_range_app.go:18]: where is `cisRangeAppDNSName` used?

File: ibm/resource_ibm_cis_range_app_test.go [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckCis(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCisRangeAppConfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCisRangeAppExists(name, &app),
+					resource.TestCheckResourceAttr(name, "origin_direct.#", "1"),
```
Comment (kavya498) [inline ibm/resource_ibm_cis_range_app_test.go:26]: Can more feature checks be added to testcase?

File: ibm/resource_ibm_cis_range_app.go (hunk 1 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_cis_range_app.go]: removed

File: ibm/resource_ibm_cis_range_app_test.go (hunk 2 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_cis_range_app_test.go:26]: added

**Pull request merged**